Minimum version is now 1.49.0?

## Expected Behavior

Compiling on Rust 1.46.0 succeeds.

## Actual Behavior

Compilation fails about some unstable feature regarding some library getting pulled in using [spinloop](https://doc.rust-lang.org/std/hint/fn.spin_loop.html), which did not stabilize until 1.49.0. This seems to be due to a newer version of a library being pulled in as part of async_graphql, even on older versions such as 2.4.6.

```
error[E0658]: use of unstable library feature 'renamed_spin_loop'
  --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/spin-0.9.0/src/relax.rs:26:9
   |
26 |         core::hint::spin_loop();
   |         ^^^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #55002 <https://github.com/rust-lang/rust/issues/55002> for more information

error: aborting due to previous error
```

## Steps to Reproduce the Problem

1. Install Rust 1.46.0 via `rustup toolchain install --minimal 1.46.0`
2. Create some project that uses async_graphql (version 2.4.6 or greater? not sure how far back it goes)
3. Attempt to compile via `cargo +1.46.0 build`

## Specifications

- Version: 2.7.4 (although this started happening for me with older versions all the way back to 2.4.6)
- Platform: Windows/Linux/Mac
- Subsystem: ???

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -17,7 +17,7 @@ jobs:
       matrix:
         include:
           - { rust: stable, os: ubuntu-latest }
-          - { rust: 1.50.0, os: ubuntu-latest }
+          - { rust: 1.51.0, os: ubuntu-latest }
     steps:
       - name: Checkout
         uses: actions/checkout@v2

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -9,8 +9,15 @@ on:
       - master
 
 jobs:
-  build:
-    runs-on: ubuntu-latest
+  tests:
+    name: Run tests - Rust (${{ matrix.rust }}) on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - { rust: stable, os: ubuntu-latest }
+          - { rust: 1.49.0, os: ubuntu-latest }
     steps:
       - name: Checkout
         uses: actions/checkout@v2
@@ -18,7 +25,7 @@ jobs:
           submodules: true
       - uses: actions-rs/toolchain@v1
         with:
-          toolchain: stable
+          toolchain: ${{ matrix.rust }}
           override: true
           components: clippy, rustfmt
       - name: Check format
@@ -34,7 +41,24 @@ jobs:
       - name: Clean
         run: cargo clean
 
-      # examples
+  examples:
+    name: Build examples - Rust (${{ matrix.rust }}) on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - { rust: stable, os: ubuntu-latest }
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.rust }}
+          override: true
+          components: clippy, rustfmt
       - name: Check examples format
         run: cargo fmt --all -- --check
         working-directory: ./examples
@@ -48,10 +72,22 @@ jobs:
         run: cargo clean
         working-directory: ./examples
 
-      # build on nightly
+  build-nightly:
+    name: Build nightly - Rust (${{ matrix.rust }}) on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - { rust: nightly, os: ubuntu-latest }
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
       - uses: actions-rs/toolchain@v1
         with:
-          toolchain: nightly
+          toolchain: ${{ matrix.rust }}
           override: true
           components: clippy, rustfmt
       - name: Build on nightly with all features

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -17,7 +17,7 @@ jobs:
       matrix:
         include:
           - { rust: stable, os: ubuntu-latest }
-          - { rust: 1.51.0, os: ubuntu-latest }
+          - { rust: 1.49.0, os: ubuntu-latest }
     steps:
       - name: Checkout
         uses: actions/checkout@v2
@@ -27,11 +27,6 @@ jobs:
         with:
           toolchain: ${{ matrix.rust }}
           override: true
-          components: clippy, rustfmt
-      - name: Check format
-        run: cargo fmt --all -- --check
-      - name: Check with clippy
-        run: cargo clippy --all
       - name: Build with all features
         run: cargo build --all-features
       - name: Build
@@ -40,6 +35,48 @@ jobs:
         run: cargo test --all --all-features
       - name: Clean
         run: cargo clean
+
+  rustfmt:
+    name: Run rustfmt - Rust (${{ matrix.rust }}) on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - { rust: stable, os: ubuntu-latest }
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.rust }}
+          override: true
+          components: rustfmt
+      - name: Check format
+        run: cargo fmt --all -- --check
+
+  clippy:
+    name: Run clippy - Rust (${{ matrix.rust }}) on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - { rust: stable, os: ubuntu-latest }
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.rust }}
+          override: true
+          components: clippy
+      - name: Check with clippy
+        run: cargo clippy --all
 
   examples:
     name: Build examples - Rust (${{ matrix.rust }}) on ${{ matrix.os }}
@@ -89,7 +126,6 @@ jobs:
         with:
           toolchain: ${{ matrix.rust }}
           override: true
-          components: clippy, rustfmt
       - name: Build on nightly with all features
         run: cargo build --all-features
       - name: Build on nightly

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -17,7 +17,7 @@ jobs:
       matrix:
         include:
           - { rust: stable, os: ubuntu-latest }
-          - { rust: 1.49.0, os: ubuntu-latest }
+          - { rust: 1.50.0, os: ubuntu-latest }
     steps:
       - name: Checkout
         uses: actions/checkout@v2

--- a/README.md
+++ b/README.md
@@ -34,7 +34,7 @@
 * [Docs](https://docs.rs/async-graphql)
 * [GitHub repository](https://github.com/async-graphql/async-graphql)
 * [Cargo package](https://crates.io/crates/async-graphql)
-* Minimum supported Rust version: 1.49.0 or later
+* Minimum supported Rust version: 1.50.0 or later
 
 ## Safety
 

--- a/README.md
+++ b/README.md
@@ -34,7 +34,7 @@
 * [Docs](https://docs.rs/async-graphql)
 * [GitHub repository](https://github.com/async-graphql/async-graphql)
 * [Cargo package](https://crates.io/crates/async-graphql)
-* Minimum supported Rust version: 1.51.0 or later
+* Minimum supported Rust version: 1.49.0 or later
 
 ## Safety
 

--- a/README.md
+++ b/README.md
@@ -34,7 +34,7 @@
 * [Docs](https://docs.rs/async-graphql)
 * [GitHub repository](https://github.com/async-graphql/async-graphql)
 * [Cargo package](https://crates.io/crates/async-graphql)
-* Minimum supported Rust version: 1.50.0 or later
+* Minimum supported Rust version: 1.51.0 or later
 
 ## Safety
 

--- a/README.md
+++ b/README.md
@@ -34,7 +34,7 @@
 * [Docs](https://docs.rs/async-graphql)
 * [GitHub repository](https://github.com/async-graphql/async-graphql)
 * [Cargo package](https://crates.io/crates/async-graphql)
-* Minimum supported Rust version: 1.46 or later
+* Minimum supported Rust version: 1.49.0 or later
 
 ## Safety
 

